MODULE.bazel

```diff
@@ -1,6 +1,46 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "fizzbee")
+
+# Go with gazelle see https://github.com/bazel-contrib/rules_go/blob/master/docs/go/core/bzlmod.md
+bazel_dep(name = "rules_go", version = "0.46.0")
+bazel_dep(name = "gazelle", version = "0.34.0")
+
+go_sdk = use_extension("@rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(version = "1.22.4")
+
+# External go deps
+go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
+go_deps.from_file(go_mod = "//:go.mod")
+
+# run "bazel mod tidy" to update these after updating go.mod
+use_repo(
+    go_deps,
+    "com_github_golang_glog",
+    "com_github_huandu_go_clone",
+    "com_github_stretchr_testify",
+    "in_gopkg_yaml_v3",
+    "net_starlark_go",
+    "org_golang_google_protobuf",
+    "org_golang_x_sys",
+)
+
+# Proto
+bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
+bazel_dep(
+    name = "protobuf",
+    version = "26.0",
```

Comment:
previously we were using 25.2 but that doesn't exist on [bazel central](https://registry.bazel.build/modules/protobuf)